python/src/main.cpp

```diff
@@ -674,6 +680,12 @@ PYBIND11_MODULE(rippled_py, m) {
         .def("xrp", &ripple::STAmount::xrp, "Converts the STAmount to an XRPAmount (if possible).")
         .def("iou", &ripple::STAmount::iou, "Converts the STAmount to an IOUAmount (if possible).")
         .def("native", &ripple::STAmount::native, "Returns True if the STAmount represents an XRP amount (the native currency).")
+        .def("get_text",
```

Comment:
This is also a function on `STBase` and should be there instead for better coverage.